tests/resources/testrepo/.gitted/worktrees/testrepo-worktree/gitdir

```diff
@@ -0,0 +1 @@
+/home/pks/Development/elego/git/libgit2-worktree/tests/resources/testrepo-worktree/.git
```

Comment:
Well, at least my implementation works with relative paths. Thought I did already fix this in my branch, seems as I've been pushing an outdated copy. Guess it's unfortunate that git recognizes those files as binary files, thus the diff didn't show it.